src/components/Quiz/QuizRadioGroup.tsx

```diff
@@ -21,6 +25,90 @@ import { Question } from "../../types"
 interface CustomRadioProps extends RadioProps {
   index: number
   label: string
+  showAnswer: boolean
+  isSelectedCorrect: boolean
+}
+
+const CustomRadio: React.FC<CustomRadioProps> = ({
+  index,
+  label,
+  showAnswer,
+  isSelectedCorrect,
+  ...radioProps
+}) => {
+  const { state, getInputProps, getRadioProps, htmlProps } =
+    useRadio(radioProps)
+
+  // Memoized values
+  const buttonBg = useMemo<string>(() => {
+    if (!state.isChecked) return "body.inverted"
+    if (!showAnswer) return "primary.base"
+    if (!isSelectedCorrect) return "error.base"
+    return "success.base"
+  }, [state.isChecked, showAnswer, isSelectedCorrect])
+
+  const primaryBaseColor = useToken("colors", "primary.base")
+
+  const focusProps: FlexProps = {
+    outline: showAnswer ? "none" : `1px solid ${primaryBaseColor}`,
+  }
+
+  const controlFocusProps: SquareProps = {
+    bg: showAnswer ? "white" : "primary.pressed",
+  }
+
+  const getRadioControlBg = (): SquareProps["bg"] => {
+    if (showAnswer) return "white"
+
+    if (state.isChecked) return "primary.pressed"
+
+    return "disabled"
+  }
+
+  const getControlLabelColor = (): TextProps["color"] => {
+    if (!showAnswer) return "white"
+
+    if (isSelectedCorrect) return "success.base"
+
+    return "error.base"
+  }
+
+  // Render CustomRadio component
+  return (
+    <chakra.label {...htmlProps} cursor="pointer" data-group w="100%">
+      <input {...getInputProps()} />
+      <Flex
+        {...getRadioProps()}
+        w="100%"
+        p={2}
+        alignItems="center"
+        bg={buttonBg}
+        color={state.isChecked ? "white" : "text"}
+        borderRadius="base"
+        _hover={{ ...focusProps, cursor: showAnswer ? "default" : "pointer" }}
+        _focus={focusProps}
+        data-group
+      >
+        <Circle
+          size="25px"
+          bg={getRadioControlBg()}
+          _groupHover={controlFocusProps}
+          _groupFocus={controlFocusProps}
+          me={2}
+        >
+          <Text
+            m="0"
```

Comment:
@wackerow ah, some of that is leftover.

Doing quotations for numerical values is actually a DX thing. Because sizing and spacing tokens are most of the time numerical, autocompletion in the IDE triggers if you do quotations. Chakra types style props with string unions.

Of course, we are in nitpick land here and it is probably considered a negligible choice of preference. 😅

@pettinarip

Reply:
Oh I was asking the same haha

Ok thanks, that makes sense. I don't have a strong opinion about this convention. Since Chakra allows both, I wouldn't mind using one or the other in the codebase.

Reply:
Valid argument... I see what you're talking about:
<img width="488" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/f5bca59d-088b-4f5b-b22d-dd0bd5babfd0">
<img width="437" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/87d89111-efc3-4f8f-acbd-70e740fd2e52">

This doesn't show up with the `{}` notation. Hm 🤔... I suppose my preference for `{}` notation mostly stems from the syntax highlighting preference 😆... I think given the argument for the autocomplete functionality, using string syntax would be the better way to go here. 

Thinking on it, I have had more than one occasion where I attempted to use a numerical token that didn't actually exist, but there was no in-IDE-warning because it wasn't a string.